source/fuzz/fuzzer_pass.cpp

```diff
@@ -180,6 +181,25 @@ uint32_t FuzzerPass::FindOrCreate32BitFloatType() {
   return result;
 }
 
+uint32_t FuzzerPass::FindOrCreateFunctionType(
+    uint32_t return_type_id, const std::vector<uint32_t>& argument_id) {
+  // Unfortunately, FindFunctionType requires us to use additional memory
```

Comment:
Nit: I would get rid of this comment, because the memory usage is negligible here and only affects the lifetime of this function.  I think it would be worth instead remarking that since fuzzerutil::FindFunctionType requires the result type id followed by all argument ids to be presented as a single sequence, some reorganization is required.

Reply:
Fixed in 2a20e6e